lib/stylesheet/manager.rb

```diff
@@ -44,6 +44,7 @@ def self.stylesheet_details(target = :desktop, media = 'all', theme_ids = :missi
     theme_ids = [theme_ids] unless Array === theme_ids
     theme_ids = [theme_ids.first] unless target =~ THEME_REGEX
     theme_ids = Theme.transform_ids(theme_ids, extend: false)
+    theme_ids = [nil] if theme_ids.empty? # Build at least one stylesheet without any theme
```

Comment:
It's still necessary here?

Reply:
Yes, because we loop over the list to generate the stylesheets

https://github.com/discourse/discourse/blob/583177dd2defea318d04d547cea02d2af9aff10e/lib/stylesheet/manager.rb#L57-L80

In the case where we want 'no themes', we still need to generate at least one stylesheet.

Reply:
@davidtaylorhq in case you're going to refactor the CSS procompile logic, we have test case for it that we don't run by default cause it's too slow (10+ seconds if I remember correctly). It should help you catch regressions:

https://github.com/discourse/discourse/blob/2e48c43546cc5b988770170927dfebe07ce5b320/spec/components/stylesheet/manager_spec.rb#L167